lib/plug/crypto/key_generator.ex

```diff
@@ -31,7 +31,7 @@ defmodule Plug.Crypto.KeyGenerator do
 
   """
   def generate(secret, salt, opts \\ []) do
-    iterations = Keyword.get(opts, :iterations, 1000)
+    iterations = Keyword.get(opts, :iterations, 1000) |> round()
```

Comment:
I think it would be best to check that it is an integer and raise accordingly. WDYT?

Reply:
Sounds good - updated accordingly.